docs/references/environment-variables.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Environment Variables
+toc_order: 2
+---
+
+# Environment Variables
+
+## Kontena Master
+
+- `MONGODB_URI`: MongoDB connection uri (required)
+- `VAULT_KEY`: secret key for the Kontena Vault (required)
+- `VAULT_IV`: initialization vector for the Kontena Vault (required)
+- `WEB_CONCURRENCY`: number of forked master api processes (default: 1)
+- `MAX_THREADS`: number of threads inside single master api process (default: 8)
```

Comment:
Definitely. For a new user who doesn't know how the process behaves (or has no experience from running Ruby app servers) having some "most likely to be fine" initial values is great.